COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,9 @@
+# llnode Collaborator Guide
+
+llnode follows 
+[Node.js Collaborator Guide](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md), 
+with the following exception:
+
+llnode Collaborators are allowed to approve their own Pull Requests if it has
+been opened for at least 7 days and there was no reviews from other
```

Comment:
I don't really like this idea. However, if this does land, please add text about requiring a passing CI run.

Reply:
This is covered by the Node.js Collaborators Guide, but I can make it explicit here too.

Reply:
Maybe just say something to the effect of "All other landing criteria still have to be met even if its self-approved." rather than repeating the criteria.

Is it really that hard to get approval on PRs here? llnode is pretty important now that its almost our default http parser, I'd hope it gets more attention than that. Could the exception be removed, and the issue of lack of active collabs be brought up to the TSC level, or equivalent?

Reply:
> Maybe just say something to the effect of "All other landing criteria still have to be met even if its self-approved." rather than repeating the criteria.

I like that. I'll make the change.

> Is it really that hard to get approval on PRs here?

Medium and large changes tend to take a month or more to get reviewed.

> llnode is pretty important now that its almost our default http parser, I'd hope it gets more attention than that.

I think you're referring to https://github.com/nodejs/llhttp. llnode is unrelated to HTTP parsing.

Reply:
Yes, I absolutely misread, sorry!